deploy-queue/src/constants.rs

```diff
@@ -4,3 +4,5 @@ pub const CONNECTION_TIMEOUT: Duration = Duration::from_secs(10);
 pub const ACQUIRE_TIMEOUT: Duration = Duration::from_secs(10);
 pub const IDLE_TIMEOUT: Duration = Duration::from_secs(10);
 pub const BUSY_RETRY: Duration = Duration::from_secs(5);
+pub const HEARTBEAT_INTERVAL: Duration = Duration::from_secs(30);
+pub const HEARTBEAT_TIMEOUT: Duration = Duration::from_secs(5 * 60); // 5 minutes
```

Comment:
We should make sure this is longer than what github has for timeouts for jobs - if a runner looses contact with github temporarily, how long until github (and the runner?) determines the job to be failed?